index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Conference Page</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="responsive-style.css">
+  <script src="script.js" defer></script>
+  <script src="mobileMenu.js" defer></script>
+</head>
+<body>
+  <section id="home">
+    <nav>
+      <a href="index.html"><img src="img/logo.png" alt="" class="cclogo"></a>
+      <img src="img/mobile_menu.png" alt="" class="hamburger">
+      <ul class="link-items">
+        <li><a href="about.html">About</a> <img src="img/close_mobile_menu.png" alt=""></li>
+        <li><a href="index.html">Program</a></li>
+        <li><a href="index.html">Join</a></li>
+        <li><a href="index.html">Sponsor</a></li>
+        <li><a href="index.html">News</a></li>
+        <li><a href="index.html">CC Campaign</a></li>
+      </ul>
+    </nav>
+    <article class="welcome">
+      <p class="hello">"Hello! Sharing World"</p>
+      <h1 class="main-heading">CREATIVE COMMONS GLOBAL SUMMIT 2015</h1>
+      <P class="joyful">A joyful festival where people from over 80 countries who create positive change gather and share their thoughts will be held in Korea in October.</P>
+      <p class="schedule">2022.10.14 (Wed) ~ 17 (Sat)</p>
+      <p class="museum">@National Museum of Korea, Art Center Nabi, etc.</p>
+    </article>
+    <article class="main-program">
+      <h3>Main Program</h3>
+      <hr class="red-main-program">
+      <div class="programs">
+      <div class="program-activity"><img src="img/ponents_icon.png" alt="" class="program-icon"> <span class="program-title">Lecture</span><p class="program-detail">Listen to the speakers from various cooutries about the messeges of sharing and opeinging.</p></span></div>
+      <div class="program-activity"><img src="img/exhibition_icon.png" alt="" class="program-icon"> <span class="program-title">CC Exhibition</span><p class="program-detail">Appreciate various creations applying CC license of artists, organized from Art Center Nabi</p></div>
+      <div class="program-activity"><img src="img/forum_icon.png" alt="" class="program-icon"> <span class="program-title">Forum</span><p class="program-detail">Have the time to share your thoughts and opinions with experts for each topic.</p></div>
+      <div class="program-activity"><img src="img/workshop_icon.png" alt="" class="program-icon"> <span class="program-title">Workshop</span><p class="program-detail">Try creating your own work using open source license rather than just watching.</p></div>
+      <div class="program-activity"><img src="img/cc-party.png" alt="" class="program-icon"> <span class="program-title">CC Ignite</span><p class="program-detail">Get opportunities to network with CC affiliate around the world, also after the summit</p></div>
```

Comment:
- [ ] This design is fine, the only thing I would like you to add as per the requirement is the `hover effect`, in the original design, I think what you see on the left is when the card has hovered, please add that for yours too.

<img width="416" alt="2" src="https://user-images.githubusercontent.com/77843050/211072252-c7f79c0f-98aa-44e3-9bea-205f2f93e036.png">